frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -276,6 +275,8 @@ class BitBox02 extends Component<Props, State> {
                 }, () => {
                     this.setPassword();
                 });
+                // show noPasswordMatch error and do NOT continue to createBackup
+                return;
```

Comment:
I double checked and that happens in current master branch, so I opened a separate PR for this https://github.com/digitalbitbox/bitbox-wallet-app/pull/1628